launch.rb

```diff
@@ -133,8 +135,15 @@ def self.restart_any_pending_work
       github_projects = Services.config_service.projects(provider_credential: Services.provider_credential)
       github_service = FastlaneCI::GitHubService.new(provider_credential: Services.provider_credential)
 
-      run_pending_github_builds(projects: github_projects, github_service: github_service)
-      enqueue_builds_for_open_github_prs_with_no_status(projects: github_projects, github_service: github_service)
+      restart_pending_builds_task = TaskQueue::Task.new(work_block: proc {
+        run_pending_github_builds(projects: github_projects, github_service: github_service)
+      })
+      @launch_queue.add_task_async(task: restart_pending_builds_task)
```

Comment:
Can we access using `self.something`?

Reply:
Ahh, I'd rather not, since I don't want to expose this queue externally. Is there a way to do that so it isn't exposed?

Reply:
Got it